rebar.config

```diff
@@ -1,12 +1,10 @@
 {erl_opts, [
     debug_info,
-    warnings_as_errors,
-    {parse_transform, lager_transform}
+    warnings_as_errors
 ]}.
 
 {deps, [
-    jsx,
-    lager
+    jsx
```

Comment:
it looks like nobody uses `jsx` in telemetry. May I kindly ask you to remove it while we are here?

```
telemetry-net$ grep -r jsx .  
./rebar.lock:[{<<"jsx">>,{pkg,<<"jsx">>,<<"2.8.2">>},0}]}.
./rebar.lock: {<<"jsx">>, <<"7ACC7D785B5ABE8A6E9ADBDE926A24E481F29956DD8B4DF49E3E4E7BCC92A018">>}]}
./rebar.config:    jsx
```